packages/mermaid/src/diagrams/er/erRenderer.js

```diff
@@ -326,7 +326,7 @@ const drawEntities = function (svgNode, entities, graph) {
       .style('text-anchor', 'middle')
       .style('font-family', getConfig().fontFamily)
       .style('font-size', conf.fontSize + 'px')
-      .text(entityName);
+      .text(entities[entityName].alias || entityName);
```

Comment:
Although it doesn't matter much in this case, it's better to always use `??` instead of `||` to avoid unexpected bugs.